doc/commenting.rst

```diff
@@ -199,7 +198,31 @@ You can optionally add a note to the field to describe the field better.
         optional double atmospheric_pressure = 1;
     }
 
-If you want to provide a reference to a DIN or to web page which helps in understanding the field you can add a reference.
+To help understanding the field, you should add a reference.
+Every OSI message should be defined properly and should have a well cited reference.
+
+**Citation style for different sources:**
+
+- Within the text, the number system is used with the number of the source in brackets [#] for mentioning.
+- We use the so called "APA style"`link <https://apastyle.apa.org/>`_ from the American Psychological Association for referencing ().
```

Comment:
Change the link into "`APA style <https://apastyle.apa.org/>`_" and remove the brackets at the end.

Reply:
Thank you for the support, will be changed.